python/common/org/python/exceptions/KeyError.java

```diff
@@ -1,6 +1,13 @@
 package org.python.exceptions;
 
 public class KeyError extends org.python.exceptions.LookupError {
+
+	public KeyError() {
+        super();
+    }
+	public KeyError(org.python.Object[] args) {
+       super(args[0].toString());
```

Comment:
So, assuming we'll fix the `args` attribute access later on, here's what I think we should do:

* add the Tuple args attribute to BaseException
* add a `getMessage()` method to BaseException that checks the size of the Tuple, and builds the proper String message
* change the constructors to set the args tuple attribute